Nuget/Ploeh.AutoFixture.AutoFakeItEasy2.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>AutoFixture.AutoFakeItEasy2</id>
+        <version>@build.number@</version>
+        <title>AutoFixture with Auto Mocking using FakeItEasy</title>
+        <authors>Nikos Baxevanis</authors>
+        <owners>Nikos Baxevanis</owners>
+        <dependencies>
+            <dependency id="AutoFixture" version="@build.number@" />
+            <dependency id="FakeItEasy" version="2.1" />
```

Comment:
We usually follow [Postel's law](https://en.wikipedia.org/wiki/Robustness_principle) when declaring dependencies. There may be users out there who, for one reason or another, can't readily upgrade from 2.0 to 2.1, and if there's no reason to exclude them, we ought to include them.